templates/events/events.ical

```diff
@@ -0,0 +1,68 @@
+{# THIS FILE MUST BE SAVED WITH LF ENDINGS ONLY, NO CRLF FOR YOU -#}
```

Comment:
but what if i wanna 🥺👉👈

Reply:
then you'd better merge my tera fork for me (for regex replace)

Reply:
I will sometime just soonTM